apps/central-scan/backend/src/interpreter.ts

```diff
@@ -267,38 +208,6 @@ export class Interpreter {
       return bmdResult;
     }
 
-    try {
-      const hmpbResult = await this.interpretHMPBFile(
-        ballotImagePath,
-        ballotImageData,
-        detectQrcodeResult.qrcode
-      );
-
-      if (hmpbResult) {
-        const { interpretation } = hmpbResult;
-        assert(interpretation && interpretation.type === 'InterpretedHmpbPage');
-
-        if (
-          this.precinctSelection.kind !== 'AllPrecincts' &&
-          interpretation.metadata.precinctId !==
-            this.precinctSelection.precinctId
-        ) {
-          timer.end();
-          return {
-            interpretation: {
-              type: 'InvalidPrecinctPage',
-              metadata: interpretation.metadata,
-            },
-          };
-        }
-        timer.end();
-        return hmpbResult;
-      }
-    } catch (error) {
-      assert(error instanceof Error);
-      debug('interpretHMPBFile failed: %s', error.message);
-    }
-
     try {
       debug(
         'assuming ballot is a HMPB that could not be interpreted (QR data: %s)',
```

Comment:
Do we want to keep these cases for invalid HMPBs around?

Reply:
I'm removing them in a later PR.